lib/tx_build/invoke_host_function.ex

```diff
@@ -2,55 +2,45 @@ defmodule Stellar.TxBuild.InvokeHostFunction do
   @moduledoc """
   Performs the following operations:
   - Invokes contract functions.
-  - Installs WASM of the new contracts.
+  - Upload WASM of the new contracts.
   - Deploys new contracts using the installed WASM or built-in implementations.
```

Comment:
Replace "installed" with "uploaded"
```suggestion
  - Upload WASM of the new contracts.
  - Deploys new contracts using the uploaded WASM or built-in implementations.
```